AuxTel/index.rst

```diff
@@ -92,7 +127,7 @@ Operations
 .. Troubleshooting
```

Comment:
The formatting of these subsections is kind off odd in the html, I think you might need some stronger separation.